### DISCUSSION TITLE
Compatibility wth ESP-12F (D1 Mini)

### DISCUSSION THREAD

**Discussion by MattiaCC93** [khoih-prog/ESP_WiFiManager#95] [Q&A] [answered]
Hi, do you think this library is compatible with this hardware model, called "ESP8266 D1 Mini V2 ESP-12F"?

If the answer is "I don't know, probably yes", I'll buy the hardware and give it a try and then tell you the answer.

Thank you

![image](https://user-images.githubusercontent.com/38031049/197498595-981c378b-cb0c-4014-af23-f0ca5a59d680.png)

**Comment by khoih-prog** [comment, marked as answer]
It's certainly supported and I also have used many of them in multiple projects. Good and reliable boards, especially you can stack up many accessories.

**Comment by MattiaCC93** [comment]
I confirm, the software works **great** on this hardware. Thank you.
